src/schema/user.js

```diff
@@ -640,24 +640,52 @@ export const resolvers = {
       }
     },
     async addBookmark(parent, args, context, info) {
+      /**
+       * Handles single or multiple additions to bookmarks.
+       *
+       * For multiple, {smaugUserId: string, bookmarks: [{materialType, string, materialId: string}]}
+       * For single, {smaugUserId: string, materialType, string, materialId: string}
```

Comment:
Jeg foreslår at alt er multiple (og så for single må man sende en med), så slipper vi for den øgede kompleksitet ved at have 2 variationer